lib/DXIL/DxilOperations.cpp

```diff
@@ -3219,6 +3207,18 @@ void OP::GetMinShaderModelAndMask(OpCode C, bool bWithTranslation,
     mask = SFLAG(Vertex);
     return;
   }
+  // Instructions: WaveMatrix_Annotate=226, WaveMatrix_Depth=227,
+  // WaveMatrix_Fill=228, WaveMatrix_LoadRawBuf=229,
+  // WaveMatrix_LoadGroupShared=230, WaveMatrix_StoreRawBuf=231,
+  // WaveMatrix_StoreGroupShared=232, WaveMatrix_Multiply=233,
+  // WaveMatrix_MultiplyAccumulate=234, WaveMatrix_ScalarOp=235,
+  // WaveMatrix_SumAccumulate=236, WaveMatrix_Add=237
+  if ((226 <= op && op <= 237)) {
+    major = 6;
+    minor = 9;
+    mask = SFLAG(Library) | SFLAG(Compute);
```

Comment:
It wasn't supposed to be part of this change, but these should be supported in node shaders too. Can you add that to the hctdb.py file around line 650?

Reply:
Either way, it should probably not be part of this PR as it would need additional tests added.
I have created a new issue to track this: #6165